scripts/main.zeek

```diff
@@ -26,52 +29,49 @@ export {
 
 	## A default logging policy hook for the stream.
 	global log_policy: Log::PolicyHook;
+
+	## Indicator of a request related to GOZI
+	redef enum HTTP::Tags += { URL_GOZIMALWARE, };
```

Comment:
Following the SQLI pattern, I suggest that this should be 'URI_GOZIMALWARE'.

Reply:
Done.